source/val/Function.h

```diff
@@ -179,6 +192,9 @@ class Function {
   /// The block that is currently being parsed
   BasicBlock* current_block_;
 
+  /// A psudo exit block that is the successor to all return blocks
+  BasicBlock exit_block_;
```

Comment:
Oh, never mind.  I see it is set to an invalid ID in the constructor.     Either way works.   I don't know if the validator has already checked the bounds on any Ids coming in, so there *might* be a risk that a module actually uses the kInvalidID.  ?  I'd have to look elsewhere in the validator.